### ISSUE TITLE
Navigation bar is not hidden

### ISSUE THREAD

**Issue by peladillas** [lukehaas/Scrollify#66]
I can not do hide the browser navigation bar, using Scrollify, someone could fix it. I'm using Chrome for andoird.
Thanks for sharing Scrollify!

**Comment by bini7a** [comment]
Scrollify has no hide method. Simply use jquery method to hide them.

**Comment by lukehaas** [comment]
@bini7a This is not correct, you can't use jQuery to hide the browser's navigation bar.

**Comment by bini7a** [comment]
@lukehaas I don't think OP really means browsers navigation bar but think site navigation bar.

**Comment by lukehaas** [comment]
@bini7a you're mistaken, OP does mean the browser's navigation bar. Most sites are able to hide this on mobile by scrolling the page slightly. 
On a page using Scrollify, this is not possible to do. All other scrolling plugins have the same issue.
